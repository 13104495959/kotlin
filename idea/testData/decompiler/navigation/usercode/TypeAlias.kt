import testData.libraries.*

fun foo(c: SimpleClassAlias) {
}