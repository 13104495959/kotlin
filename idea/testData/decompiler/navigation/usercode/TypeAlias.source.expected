 main.kt
typealias <1>SimpleClassAlias = SimpleClass
